packages/worker/native/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// NOTE: this interface is duplicated in the client module `packages/client/src/index.ts`
+// NOTE: this interface is duplicated in the client module `packages/client/src/index.ts` for lack of a shared library
```

Comment:
Any reason not to just make a lower dep they can share?

Reply:
I will at some point, I just didn't want to add to this PR.